2.5/Dockerfile

```diff
@@ -20,13 +20,12 @@ building and running various Ruby $RUBY_VERSION applications and frameworks. \
 Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
 It has many features to process text files and to do system management tasks (as in Perl). \
 It is simple, straight-forward, and extensible." \
-    PATH=$PATH:/opt/rh/$NODEJS_SCL/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/bin:/opt/rh/$NODEJS_SCL/root/usr/bin \
-    LD_LIBRARY_PATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/lib64:/opt/rh/$NODEJS_SCL/root/usr/lib64:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/lib64 \
-    X_SCLS=$X_SCLS:rh-ruby$RUBY_SCL_NAME_VERSION \
-    MANPATH=/opt/rh/$NODEJS_SCL/root/usr/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/man:/share/man/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share/man:/opt/rh/$NODEJS_SCL/root/usr/share/man \
-    PYTHONPATH=/opt/rh/$NODEJS_SCL/root/usr/lib/python2.7/site-packages \
-    XDG_DATA_DIRS=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/share:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/share \
-    PKG_CONFIG_PATH=/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/local/lib64/pkgconfig:/opt/rh/rh-ruby$RUBY_SCL_NAME_VERSION/root/usr/lib64/pkgconfig
+    PATH="/opt/rh/rh-ruby${RUBY_SCL_NAME_VERSION}/root/usr/local/bin:/opt/rh/rh-ruby${RUBY_SCL_NAME_VERSION}/root/usr/bin:/opt/rh/${NODEJS_SCL}/root/usr/bin:${PATH}" \
+    LD_LIBRARY_PATH="/opt/rh/rh-ruby${RUBY_SCL_NAME_VERSION}/root/usr/local/lib64:/opt/rh/rh-ruby${RUBY_SCL_NAME_VERSION}/root/usr/lib64:/opt/rh/${NODEJS_SCL}/root/usr/lib64" \
+    X_SCLS="${NODEJS_SCL} rh-ruby${RUBY_SCL_NAME_VERSION}" \
```

Comment:
The drop of X_SCLS does not allow for any other SCL to be enabled before this / with this / container image build, right?

_Note: there was `:` by mistake. Shouldn't `scl_source` or `scl` throw an error somewhere? Is some `scl` check possible?_

Reply:
Not sure I follow -- having `X_SCLS` defined like this does not prevent you to enable more SCLs later, why should it? Obviously, it could be problem to have some more SCLs enabled before in the ruby container, but that is just hypothetical not really probable scenario. The goal of this was not to entirely duplicate what `enable` scriptlet does when running `scl enable`, but to define the environment so that it is the same as if we ran the `scl enable`.

Reply:
You're right. It's better this way.